### DISCUSSION TITLE
Very inconsistent data

### DISCUSSION THREAD

**Discussion by Imposible-to-find-a-username** [teslamate-org/teslamate#2708] [Q&A]
I managed to get Teslamate running on my Synologi NAS but the data I see is very inconsistent and not very usefull. I have installed/re-installed/restarted hundreds of times without any change. Some drives are not even logged, and it seems the car is not consistently connected somehow.
What can be the culprit here?

![image](https://user-images.githubusercontent.com/106878632/174608557-b480330e-7df1-4d5e-b459-475153eccc98.png)

![image](https://user-images.githubusercontent.com/106878632/174608786-2586f51b-835a-4562-ab0f-69528d393f06.png)

**Comment by dominikmeier** [comment]
It is highly likely that this is not a teslamate nor a Synology issue. I have the same behavior when I'm crossing borders and my Tesla is loosing cell phone connection for about 12 Minutes. Therefore nothing will be logged during this time. This is all caused of poor Network roaming. Perhaps you can check if your Tesla is loosing cell phone reception during the ride?
